packages/storybook/src/stories/ino-tab-bar/ino-tab-bar.stories.ts

```diff
@@ -46,6 +47,12 @@ Playground.argTypes = {
     },
     options: [0, 1, 2, 3],
   },
+  autoFocus: {
```

Comment:
We don't have to explicitly declare the control for primitive types like `boolean`, `number`, and `string`. This can be safely removed.